sdk/php/runtime/template/README.md

```diff
@@ -311,3 +311,48 @@ class Example
      // ...
 }
 ```
+
+#### Directories and Files
+
+Directories and Files have some additional metadata that can be specified
+
+##### Defaults
+
+Specifying a default value for a `Directory` or `File` isn't easily possible. 
+Instead you can specify a `DefaultPath` attribute with an absolute or relative path.
+If relative then the path is relative to the `source` specified in the `dagger.json` file.
+
+This tells Dagger; if no argument is specified, fetch the `Directory` or `File` from the `DefaultPath`.
+
+The following example shows how you would get the parent directory of your `source`:
+
+```php
+#[DaggerFunction]
+public function myDaggerFunction(
+    #[DefaultPath('../')]
```

Comment:
With https://github.com/dagger/dagger/pull/8391, it's now `.`

Reply:
From what I understood `.` is the source dir.

My example is to get the parent of the source dir.

Do you think it would be helpful for me to demonstrate both?

Reply:
I now demonstrate both and then link to the docs for further detail.

https://github.com/dagger/dagger/blob/d08ed252db2e8bf2fd340450c8cacfbab4a1e1b3/sdk/php/runtime/template/README.md?plain=1#L330-L354

Reply:
> From what I understood . is the source dir.

Nope, `.` is the location of the `dagger.json` dir, not the source dir

> My example is to get the parent of the source dir.

I think you should also add an example for absolute path.